Makefile

```diff
@@ -7,7 +7,8 @@ goarch=amd64
 ldXflags=github.com/mongodb/mongodbatlas-cloudformation-resources/util.defaultLogLevel=info
 ldXflagsD=github.com/mongodb/mongodbatlas-cloudformation-resources/util.defaultLogLevel=debug
 
-MOCKERY_VERSION=v2.38.0
+MOCKERY_VERSION=v2.42.1
+GOLANGCI_VERSION=v1.60.3 # Also update golangci-lint GH action in code-health.yml when updating this version
```

Comment:
this variable was missing, so it was always getting the latest linter version in "make tools", so local linter was not the same as in codehealth GHA.   use same linter version and config as in TF